plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManager.kt

```diff
@@ -49,20 +49,22 @@ internal class LocationPuckManager(
     }
 
   fun initialize(style: StyleInterface) {
```

Comment:
Initialize method called from updateSettings waits for the style at [getStyle](https://github.com/mapbox/mapbox-maps-android/blob/b2f3a56194ef54b93eddde69656fccac648d87ae/plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManager.kt#L96). 
If updateSettings was called multiple times multiple callbacks will fire after style arrives causing crash.